typed-racket-lib/typed-racket/typecheck/tc-structs.rkt

```diff
@@ -517,7 +512,7 @@
   (define sty (mk/inner-struct-type names desc parent-type))
 
   (register-sty! sty names desc)
-  (register-struct-bindings! sty names desc #f)
+  (void (register-struct-bindings! sty names desc #f))
```

Comment:
This may need a comment (but I'm not sure how to word it) because,

- first thought: `register-struct-bindings!` must return `(void)` because of the `!`
- second thought: this must be a bug, because we're ignoring a list of bindings

It took a while for me to realize that this function `tc/builtin-struct` runs at a different time than the similar-named `tc/struct`. The latter runs during type checking and helps make the `#%type-decl` module. This function appears in the `#%type-decl` module and so cannot use the list of bindings.